playlists/excel.yaml

```diff
@@ -496,10 +496,19 @@
   group: Preview APIs
   api_set:
     ExcelApi: 1.7
+- id: excel-events-tablecollection-changed
+  name: Events - Table collection changed
+  fileName: events-tablecollection-changed.yaml
+  description: Add event handlers for table collection onDataChanged event
```

Comment:
If it's just one event handler, suggest you change "event handlers" to "event handler".